src/status_im2/contexts/onboarding/profiles/style.cljs

```diff
@@ -48,11 +48,10 @@
 (def login-profile-card
   {:margin-bottom 20})
 
-(def info-message
-  {:margin-top 8})
-
-(def forget-password-button
-  {:margin-vertical 8})
+(def error-message
+  {:margin-top     8
+   :flex-direction :row
+   :align-items    :center})
 
 (defn login-button
```

Comment:
I think a change is still mising, the loggin-button margin bottom should be 12, currently is 20, wdyt?